ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -0,0 +1,118 @@
+    weak var balanceTitle: UILabel!
+    weak var reloadButton: UIButton!
+    weak var accountButton: UIButton!
+    
+    override init() {
+        self.viewModel = BalanceViewModel()
+        self.rm = RequestManager()
+        super.init()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.view.backgroundColor = UIColor.Theme.backgroundColor
+        
+        let screenStackView = UIStackView()
+        screenStackView.axis = .vertical
+        screenStackView.spacing = 60
+        self.screenStackView = screenStackView
+        self.view.addSubview(screenStackView)
+        
+        screenStackView.snp.makeConstraints { make in
+            make.centerX.centerY.equalToSuperview()
+            make.leading.greaterThanOrEqualTo(20)
+            make.trailing.lessThanOrEqualTo(-20)
+        }
+        
+        setupBalanceField()
+        setupButtonsStack()
+    }
+    
+    fileprivate func setupBalanceField() {
+        let balanceTitle = UILabel()
+        balanceTitle.text = L10n.Balance.title
+        balanceTitle.textColor = UIColor.Theme.labelBlue
+        balanceTitle.textAlignment = .center
+        balanceTitle.font = UIFont.systemFont(ofSize: 18)
+        self.balanceTitle = balanceTitle
+        screenStackView.addArrangedSubview(balanceTitle)
+        
+        let balanceLabel = UILabel()
+        #warning("TODO")
+        balanceLabel.text = "1357 Kč"
+        balanceLabel.textColor = UIColor.Theme.textColor
+        balanceLabel.adjustsFontSizeToFitWidth = true
+        balanceLabel.textAlignment = .center
+        balanceLabel.font = UIFont.boldSystemFont(ofSize: 80)
+        self.balanceLabel = balanceLabel
+        screenStackView.addArrangedSubview(balanceLabel)
+    }
+    
+    fileprivate func setupButtonsStack() {
+        let buttonsStackView = UIStackView()
+        screenStackView.addArrangedSubview(buttonsStackView)
+        
+        let reloadButton = UIButton()
+        reloadButton.setImage(Asset.reloadIcon.image, for: .normal)
+        self.reloadButton = reloadButton
+        buttonsStackView.addArrangedSubview(reloadButton)
+        
+        let spacerView = UIView()
+        buttonsStackView.addArrangedSubview(spacerView)
+        
+        let accountButton = UIButton()
+        accountButton.setImage(Asset.accountIcon.image, for: .normal)
+        accountButton.isUserInteractionEnabled = true
+        self.accountButton = accountButton
+        buttonsStackView.addArrangedSubview(accountButton)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        accountButton.addTarget(self, action: #selector(accountBtnHandle), for: .touchDown)
+        reloadButton.addTarget(self, action: #selector(reloadBalance), for: .touchDown)
```

Comment:
Button vždy .touchUpInside, pokud to nemá speciální důvod. Uživatelé jsou zvyklí, že zmáčknou, rozmyslí si to, odjedou prstem z buttonu, zvednou ho někde vedle a nic se nestane. U tebe zmáčknou a než stihnout zvednout prst, tak už letí request.

Reply:
opraveno